tests/test_attachments.py

```diff
@@ -270,3 +270,48 @@ async def default_handler(message: IncomingMessage, bot: Bot) -> None:
     # - Assert -
     assert incoming_message
     assert incoming_message.file == domain_attachment
+
+
+API_UNSUPPORTED_FILE_ATTACHMENT_TYPE = (
+    {
+        "data": {
+            "id": "9657694f-166e-575a-b6ea-6575521d4f57",
+            "pack": "2c00198f-6efb-5b36-bed1-1a380bc85710",
+            "version": 1637674405,
+        },
+        "type": "sticker",
```

Comment:
If we add stickers, this test will fail. Use
```json
{
  "data": {"foo": "bar"},
  "type": "baz"
}
```

Reply:
Fix.